### ISSUE TITLE
Deploying additional nodes after initial deployment

### ISSUE THREAD

**Issue by Venoox** [onedr0p/flux-cluster-template#588]
If I want to deploy more nodes, do I just need to add them to the config and run the whole process again?

**Comment by onedr0p** [comment]
Yes that's pretty much it. Make sure you run the cluster prepare and cluster installation Ansible playbooks. However it's always a good to have application data backed up before continuing.
